### ISSUE TITLE
Use PowerShell icon for terminals on Windows

### ISSUE THREAD

**Issue by dhirschfeld** [jupyterlab/jupyterlab#3717]
Since the default terminal is PowerShell it might be nice to indicate that with the icon:

<img src=https://raw.githubusercontent.com/PowerShell/PowerShell/master/assets/Powershell_256.png>

From the [PowerShell repo](https://github.com/PowerShell/PowerShell/tree/master/assets) reusing the icon would seem to fall under the permitted uses.

**Comment by blink1073** [comment]
I don't think we can tell for a fact which shell the user is running from the front end.  We also can't tell for a fact which OS the notebook server is running on (though we could add that functionality).  I personally would be confused if I launched bash (which I do on Windows) and the icon were for PowerShell.

**Comment by dhirschfeld** [comment]
Out of interest, do you use bash under WSL?

Maybe the icon could be configurable?

Would also be very nice to be able to configure multiple terminals - jupyter/notebook#3262

I'd like to provide our users both PowerShell & Bash too! 😃

**Comment by blink1073** [comment]
Yep, configurability is a good option.  I use Git for Windows, which comes with bash.  I set the `SHELL` environment variable to point to that executable so terminado picks it up.
